node/node.go

```diff
@@ -82,9 +84,9 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	if conf.GRPC.BasicAuthCredential != "" {
 		enableHTTPAuth = true
 	}
-
-	httpServer := http.NewServer(conf.HTTP, enableHTTPAuth)
 	grpcServer := grpc.NewServer(conf.GRPC, st, syn, net, consMgr)
+	httpServer := http.NewServer(conf.HTTP, enableHTTPAuth)
+	jsonrpcServer := jsonrpc.NewServer(conf.JSONRPC, enableHTTPAuth) // TODO: enable auth???
```

Comment:
Don't need to enable auth, because you need transcode json-rpc context to grpc metadata.

Please remove flag `enableHTTPAuth`.

Reply:
https://github.com/pactus-project/pactus/pull/1183/commits/a89973cbef09b33c059d9701b08897ecbfc08cda